packages/frontend-2/components/projects/MoveToWorkspaceDialog.vue

```diff
@@ -2,41 +2,45 @@
   <LayoutDialog v-model:open="open" max-width="sm" :buttons="dialogButtons">
     <template #header>Move project to workspace</template>
     <div class="flex flex-col space-y-4">
-      <template v-if="!isCreatingWorkspace">
-        <ProjectsWorkspaceSelect
-          v-if="hasWorkspaces"
-          v-model="selectedWorkspace"
-          :items="workspaces"
-          :disabled-roles="[Roles.Workspace.Member, Roles.Workspace.Guest]"
-          disabled-item-tooltip="Only workspace admins can move projects into a workspace."
-          label="Select a workspace"
-          help="Once a project is moved to a workspace, it cannot be moved out from it."
-          show-label
+      <template v-if="!workspace">
```

Comment:
This whole part will be remove from this modal soon, when workspace selection will happen outside of the modal and we will always pass a workspaceId as prop